doc/building.md

```diff
@@ -1285,10 +1285,10 @@ at least the following targets are known to work:
 | ppc64le-linux-gnu        |
 | s390x-linux-gnu          |
 
-`BASE_OS` must be one of "OEL6" for Oracle Enterprise Linux 6 or "Fedora" (if
-not specified "OEL6" will be the default). If the base OS is "Fedora" the
+`BASE_OS` must be one of `OL` for Oracle Enterprise Linux or `Fedora` (if
+not specified `OL` will be the default). If the base OS is `Fedora` the
 corresponding Fedora release can be specified with the help of the
-`BASE_OS_VERSION` option (with "27" as default version). If the build is
+`BASE_OS_VERSION` option (with `27` as default version). If the build is
```

Comment:
Also unrelated but maybe worth fixing: the default Fedora version is 41 nowadays.